timed/subscription/views.py

```diff
@@ -50,6 +56,64 @@ class OrderViewSet(viewsets.ModelViewSet):
         | IsAuthenticated & IsReadOnly
     ]
 
+    def create(self, request, *args, **kwargs):
+        """Override so we can issue emails on creation."""
+        # check if order is acknowledged and created by admin/accountant
+        if (
+            request.method == "POST"
+            and request.data.get("acknowledged")
+            and not (request.user.is_accountant or request.user.is_superuser)
+        ):
+            raise ValidationError("User can not create confirmed orders!")
+
+        template_txt = get_template("notify_accountants_order.txt")
```

Comment:
Refactor: I suggest moving the whole emailing code into it's own module or at least a separate function/method.

Reply:
done